devsetup/Makefile

```diff
@@ -156,8 +156,7 @@ edpm_compute: ## Create EDPM compute VM
 
 .PHONY: edpm_compute_repos
 edpm_compute_repos: ## Configure EDPM compute VM with repositories
-	$(eval $(call vars))
-	scripts/edpm-compute-repos.sh ${EDPM_COMPUTE_SUFFIX}
+	@echo "DEPRECATED: this target has no effect and will be removed. Repo setup is done by the 'repo-setup' ansible service"
```

Comment:
Maybe this should also exit with an error, opinions welcome

Reply:
an error would be nice, it would catch the attention of people who are running it inside a script

Reply:
This can't happen yet though because CI jobs call this, so this will need to land first then a ci-framework change will need to stop calling this target